src/main/kotlin/dev/stalla/builder/validating/ValidatingAtomBuilder.kt

```diff
@@ -13,17 +13,14 @@ internal class ValidatingAtomBuilder : AtomBuilder {
     private var contributorBuilders: MutableList<PersonBuilder> = mutableListOf()
     private var linkBuilders: MutableList<LinkBuilder> = mutableListOf()
 
-    override fun addAuthorBuilder(authorBuilder: PersonBuilder): AtomBuilder = apply {
-        authorBuilders.add(authorBuilder)
-    }
+    override fun addAuthorBuilder(authorBuilder: PersonBuilder): AtomBuilder =
+        apply { authorBuilders.add(authorBuilder) }
```

Comment:
For consistency, I've refactored these as well to new line expression bodies, even though the lines have not been too long here. Preferred the previous style more though.